discourse/janitor.json

```diff
@@ -22,10 +22,16 @@
       "label": "Discourse",
       "proxy": "https",
       "preview": true
+    },
+    "1080": {
```

Comment:
Nit: In the future, please sort ports by increasing order (i.e. `"1080"` between `"22"` and `"8088"`).